jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -140,6 +140,11 @@ then
 elif [[ "${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_centos" ]]
 then
   make feature_tests
+elif [[ "${TESTS_FOR}" == "e2e_tests" ]]
+then
+  pushd "/home/${USER}/tested_repo"
```

Comment:
you have a var, why not to use it 😊
```suggestion
  pushd "${CAPM3PATH}"
```

Reply:
Done